### DISCUSSION TITLE
Cant find any website API documentation

### DISCUSSION THREAD

**Discussion by pyranota** [pablouser1/ProxiTok#175] [Q&A] [answered]
Is there any API? How can i get, lets say, download link for video with `x` id in code? The only way to do this i see, is just to scrap it from Proxitok html page. Am I wrong? Is this the only way?

**Comment by NoPlagiarism** [comment, marked as answer]
Hey, I guess you better use normal scrapers, as said in #84

https://github.com/pablouser1/ProxiTok/issues/84#issuecomment-1269766836

**Reply by pyranota** [reply]
Oh, ok. Im writing simple private tiktok frontend for android. And i dont really want to create and host any kind of proxiserver and using tiktok api directly will kill all privacy. So as said, its probably ok to scrape proxitok. If its the only way, i would optimize my code as much as possible to avoid unnecessary proxitok requests.

**Reply by pyranota** [reply]
Thank you for the link on the discussion, btw

**Reply by NoPlagiarism** [reply]
> Im writing simple private tiktok frontend for android... So as said, its probably ok to scrape proxitok

Good Luck with that \:D

Don't forget to add instances switch support with custom instances 

And, for machine understandable instances list, see [Farside](https://github.com/benbusby/farside/blob/main/services.json#L405) \[[CloudFlare version](https://github.com/benbusby/farside/blob/main/services-full.json#L438)\] or [mine with smaller size](https://github.com/NoPlagiarism/frontend-instances-list/tree/master/instances/tiktok/proxitok). And, ofc, you can scrape it from [Wiki](https://github.com/pablouser1/ProxiTok/wiki/Public-instances)
